app/src/main/java/app/insti/utils/TagCategories.java

```diff
@@ -0,0 +1,33 @@
+package app.insti.utils;
+
+/**
+ * Created by Shivam Sharma on 13-08-2018.
+ */
+
+public class TagCategories {
+
+    public static final String[] CATEGORIES = new String[]{
+            "Used Flexes",
+            "Plastic Bottles (in Hostel messes)",
+            "Placards on Trees",
+            "Request for donation of clothes",
+            "Other donations",
+            "Cattle Issues",
+            "Autorickshaws",
+            "Potholes in Roads",
+            "Broken stormwater drains",
+            "Desilting - lakes",
+            "Flooding of roads and footpaths",
```

Comment:
**IMPORTANT**
Why are these hard coded here? We cannot release an update if this list needs to be changed!

Reply:
We are working on the dynamic tags API for version 2. But for time being leave it as it is

Reply:
Put these in a JSON file and upload it as a static file to the server instead then. This really is no place for such a bunch of user-facing strings.

Reply:
For time being, these are all temporary. Once the ML model kicks in, we won't be needing these. And I have to make an API for explicitly calling it on each character change.

Reply:
I won't pursue this then, but make sure these are proof read. Like I said, there _cannot_ be an update for a change in this.

Reply:
Yeah sure. Thanks. So I'll leave this as it is.